mautrix_facebook/user.py

```diff
@@ -255,6 +255,13 @@ async def load_session(self, _override: bool = False, _raise_errors: bool = Fals
         if self._is_logged_in and not _override:
             return True
         elif not self.state:
+            # If we have a user in the DB with no state, we can assume
+            # FB logged us out and the bridge has restarted
+            await self.push_bridge_state(
+                BridgeStateEvent.BAD_CREDENTIALS,
+                error="fb-auth-error",
+                message="Bridge has restarted in BAD_CREDENTIALS state. Please delete and log in again.",
```

Comment:
The error message isn't optimal for showing to users, this might be better (message will be "You're not logged into Messenger"):

```suggestion
                error="logged-out"
```

alternatively a new error code can be added (and the corresponding message should go to `BridgeState.human_readable_errors` at the top of this file)